CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 5.1.0 - 2020-08-20
+### Added
+- Forward merge release/4.x
```

Comment:
Iets meer info hier zou wel fijn zijn (om welke versies en/of features gaat het)